### ISSUE TITLE
Key Error trying to process implementation defined server errors

### ISSUE THREAD

**Issue by gschmottlach-xse** [pengutronix/aiohttp-json-rpc#24]
It looks like exceptions.py does not handle implementation defined server errors (e.g. JSON-RPC errors in the range -32000 to -32099).  On line ```exceptions.py:67```

```
return _EXCEPTION_LOOKUP_TABLE[error_code]
```

There is no mapping for error codes in the range -32000 to -32099. One of the services I communicate with over JSON-RPC return implementation defined server errors and my application throws a KeyError when such an error is returned. Perhaps bit of logic could be added to exceptions.py to map these implementation defined server errors to a new generic ```RpcServerDefinedError```?

**Comment by fscherf** [comment]
Hi @gschmottlach-xse :)
First of all: Thx for using und reporting problems!

You are right, application specific error codes are not implemented yet, simply because no one (especially me) needed it before. But the code is prepared for this feature.

`aiohttp_json_rpc/exceptions.py:37`
```python
_EXCEPTION_LOOKUP_TABLE = {                           
    e.ERROR_CODE: e for e in RpcError.__subclasses__()
}
```

The idea of this snippet was to implement custom errors like this:

```python
# server.py

from aiohttp_json_rpc import JsonRpc, RpcError

class ComputerSaysNoError(RpcError):
    ERROR_CODE = -32017
    MESSAGE = 'computer says no'


if __name__ == '__main__':
    rpc = JsonRpc()

    [...]
```

I like your idea to throw a generic `RpcServerDefinedError` if no one defines a error with the given error code.
Feel free to pull-request, or you wait for me to implement it (will take me a few days to find time, but it's simple task i think)

**Comment by gschmottlach-xse** [comment]
Thanks for looking into this. Assuming you can bang something out in a few days, I will patient ;-)

I am using your module in some sample code that I want to share with colleagues. Ideally, knowing that your package is a dependency, they'll ultimately just do a ```pip install ...``` to get these updates. That's why I've been reluctant to fork it until now since it's easier for them to go the ```pip install`` route rather than clone an alternate version.

**Comment by fscherf** [comment]
Sorry it took me so long. Much work, was sick two times, and it was a bit more complicated than i thougt.
I pushed the changes and i'll make a release very soon.

**Comment by fscherf** [comment]
fixed in v0.10
